AlphaWallet/Settings/Views/SettingTableViewCell.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  SettingTableViewCell.swift
+//  AlphaWallet
+//
+//  Created by Nimit Parekh on 06/04/20.
+//
+
+import UIKit
+
+protocol SettingTableViewCellDelegate: class {
+    func onOffPassCode(cell: SettingTableViewCell, switchState isOnOff: Bool)
+}
+
+class SettingTableViewCell: UITableViewCell {
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+    }
+    weak var delegate: SettingTableViewCellDelegate?
+    fileprivate var titleTopConstraint: NSLayoutConstraint?
+
+    var settings: SettingModel? {
+        didSet {
+            guard let settingItem = settings else { return }
+            if let title = settingItem.title {
+                settingTitle.text = title
+            }
+            if let subtitle = settingItem.subTitle {
+                settingSubTitle.text = subtitle
+                let subTitleLength = subtitle.count
+                if subTitleLength > 0 {
+                    titleTopConstraint = settingTitle.centerYAnchor.constraint(equalTo: self.containerView.centerYAnchor, constant: -10)
+                    titleTopConstraint?.isActive = true
+                }
+            }
+            if let icon = settingItem.icon {
+                settingIconImage.image = icon
+            }
+        }
+    }
+    let containerView: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.clipsToBounds = true // this will make sure its children do not go out of the boundary
+        return view
+    }()
+    
+    let settingIconImage: UIImageView = {
```

Comment:
Most of these properties should be private

Reply:
Done